src/sentry/eventstore/reprocessing/redis.py

```diff
@@ -0,0 +1,166 @@
+import uuid
+from collections.abc import Sequence
+from datetime import datetime
+from typing import Any
+
+import redis
+from django.conf import settings
+
+from sentry.utils import json
```

Comment:
I see that you're only using `dumps`, explicit importing this would be better:

```suggestion
from sentry.utils.json import dumps
```